src/libs/ReportUtils.ts

```diff
@@ -516,25 +712,21 @@ function shouldDisableDetailPage(report) {
 
 /**
  * Returns true if this report has only one participant and it's an Expensify account.
- * @param {Object} report
- * @returns {Boolean}
  */
-function isExpensifyOnlyParticipantInReport(report) {
-    const reportParticipants = _.without(lodashGet(report, 'participantAccountIDs', []), currentUserAccountID);
-    return reportParticipants.length === 1 && _.some(reportParticipants, (accountID) => _.contains(CONST.EXPENSIFY_ACCOUNT_IDS, accountID));
+function isExpensifyOnlyParticipantInReport(report: OnyxEntry<Report>): boolean {
+    const reportParticipants = report?.participantAccountIDs?.filter((accountID) => accountID !== currentUserAccountID) ?? [];
+    return reportParticipants.length === 1 && reportParticipants.some((accountID) => CONST.EXPENSIFY_ACCOUNT_IDS.includes(accountID));
 }
 
 /**
  * Returns whether a given report can have tasks created in it.
  * We only prevent the task option if it's a DM/group-DM and the other users are all special Expensify accounts
  *
- * @param {Object} report
- * @returns {Boolean}
  */
-function canCreateTaskInReport(report) {
-    const otherReportParticipants = _.without(lodashGet(report, 'participantAccountIDs', []), currentUserAccountID);
+function canCreateTaskInReport(report: OnyxEntry<Report>): boolean {
+    const otherReportParticipants = report?.participantAccountIDs?.filter((accountID) => accountID !== currentUserAccountID);
```

Comment:
```suggestion
    const otherReportParticipants = report?.participantAccountIDs?.filter((accountID) => accountID !== currentUserAccountID) ?? [];
```

Reply:
is this needed ? `otherReportParticipants?.` is resolving that so I think we dont need that default value